### DISCUSSION TITLE
#【分享】#各阵营图标，想要的可以自取呀

### DISCUSSION THREAD

**Discussion by mwk07** [share121/inter-knot#980] [其他]
![7df7eee0-725b-4c7e-8536-944bb14d2fb0](https://github.com/user-attachments/assets/7009b5c2-1807-4187-aa44-a0b83ecd0eab)
![2cc0dd2628c722e73004a19b84ab1f99_2079635245468948683](https://github.com/user-attachments/assets/a5bec050-149a-4801-b0b6-e7a3de95ec9c)
![5e2833934e34b266dde548849040d433_7055688299901497327](https://github.com/user-attachments/assets/023c44ec-1f85-4254-aa0f-1e7b98e9df86)
![7eda813e089f1bdcec4e7c7e93b101d6_6078806162773208785](https://github.com/user-attachments/assets/6ee1b0df-dbf2-43ba-b091-6a4f576aefb9)
![193ce3eb96bce41fc84619164cd3aed5_748736223171964552](https://github.com/user-attachments/assets/59d52946-f7f3-43ed-94cc-e0de8ccb5121)
![dc54e0eb8b6ba749e3bc6219dabbb6b1_4606263158565041411](https://github.com/user-attachments/assets/a2efcc6c-5331-4343-b2f9-eabf7b83c0f3)

**Comment by gaojiaobei** [comment]
楼主厉害！

**Comment by minyue-yu** [comment]
绝区零官网可用直接拿到

**Comment by silverwolf510trillion** [comment]
好人一生18胞胎！
